src/batch_group/entity.rs

```diff
@@ -58,6 +58,11 @@ impl BatchGroup {
         self.events
             .push(BatchGroupEvent::BatchGroupDescriptionUpdated { description });
     }
+
+    pub fn update_config(&mut self, config: BatchGroupConfig) {
```

Comment:
lets check here if the config is actually new before adding the event. Same with the other mutation (descriptions). Also add this check to Address please.